docs/frontity-cli/dev.md

```diff
@@ -19,6 +19,25 @@ npx frontity dev [options]
 |     `--target <target>`      | create bundles with "es5" or "module". Default target is "module".                                                                 |
 | `-h`, `--help`  | Output usage information                                                                                                             |
 
+##### The `--production` option
+
+This flag correspond to [webpack’s mode parameter](https://webpack.js.org/configuration/mode/) so it will run webpack in the production mode as described [there](https://webpack.js.org/configuration/mode/) before launching the development server.
+
+So, if you do:
+
+```
+npx frontity dev --production
+```
+
+The webpack bundler internally will:
```

Comment:
This is not an exhaustive list so I'd say something like `it will do things like...`.